tests/integration/devfile/cmd_devfile_delete_test.go

```diff
@@ -12,8 +11,19 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+// deleteLocalConfig helps user to delete local config files with flags
+func deleteLocalConfig(args ...string) {
+	helper.CmdShouldFail("odo", args...)
```

Comment:
It's a bit unclear as to why this should fail. To get the context, I scrolled down to where the function is called. IMO, it makes more sense to limit the scope of this function to the context where it's being called. That would still get the task done and avoid any confusion for someone who's looking at using this later.

Reply:
> It's a bit unclear as to why this should fail.

oops... my bad. I should have just add a single comment so that it doesn't confuse everyone.

> To get the context, I scrolled down to where the function is called. IMO, it makes more sense to limit the scope of this function to the context where it's being called. That would still get the task done and avoid any confusion for someone who's looking at using this later.

Yes, I agree the earlier format will also get the task done. But from cleanup and customisation point of view and keeping the context also in mind, I would say this approach would be better if we add a single line of comment why the `CmdShouldFail` or Pass. And going forward someone will just use this function according to their need. Also this function has reduced number of lines from the test scripts.

Reply:
BTW this function ```deleteLocalConfig``` validate both pass and fail condition in a single function block.

Reply:
@prietyc123 Can you please move it to helper package for re usability purpose